yt/utilities/answer_testing/answer_tests.py

```diff
@@ -44,46 +43,25 @@ def parentage_relationships(ds):
     return result 
 
 def grid_values(ds, field):
-    # The hashing is done here so that there is only one entry for
-    # the test that contains info about all of the grids as opposed
-    # to having a separate 'grid_id : grid_hash' pair for each grid
-    # since that makes the answer file much larger
-    result = None
+    result = {}
```

Comment:
I'm not an expert on hashing, but good job here ! Your dict-based refactor is much more readable.